pkg/serverless/trace/trace.go

```diff
@@ -0,0 +1,65 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
+// under the Apache License Version 2.0.
+// This product includes software developed at Datadog (https://www.datadoghq.com/).
+// Copyright 2016-present Datadog, Inc.
+
+package trace
+
+import (
+	"context"
+
+	"github.com/DataDog/datadog-agent/pkg/trace/agent"
+	"github.com/DataDog/datadog-agent/pkg/trace/config"
+	"github.com/DataDog/datadog-agent/pkg/util/log"
+)
+
+// ServerlessTraceAgent represents a trace agent in a serverless context
+type ServerlessTraceAgent struct {
+	ta     *agent.Agent
+	cancel context.CancelFunc
+}
+
+// Load abstracts the file configuration loading
+type Load interface {
+	Load() (*config.AgentConfig, error)
+}
+
+// LoadConfig is implementing Load to retrieve the config
+type LoadConfig struct {
+	Path string
+}
+
+// Load loads the config from a file path
+func (l *LoadConfig) Load() (*config.AgentConfig, error) {
+	return config.Load(l.Path)
+}
+
+// Start starts the agent
+func (s *ServerlessTraceAgent) Start(enabled bool, loadConfig Load, waitingChan chan bool) {
```

Comment:
I'm not sure if this needs to be a whole class. It seems like a stand alone function to load/start the agent would do the same thing.